katcp/resource_client.py

```diff
@@ -1114,16 +1124,37 @@ def wait(self, sensor_name, condition_or_value, timeout=5):
             sensor is not present
 
         """
+        if quorum is None:
+            quorum = len(self.clients)
+        elif quorum < 1:
+            quorum = int(math.ceil(quorum * len(self.clients)))
```

Comment:
Indeed. Or you can pick 0.999 as it rounds up :-)

I've improved the docstring wording in a pending commit.